modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,35 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+== packagemanifests
+`run packagemanifests` subcommand deploys an Operator's package manifests with OLM. The command argument must be set to a valid package manifest root directory, for example, ``<project-root>/packagemanifests`.
+
+.`packagemanifests` arguments
 [options="header",cols="1,3"]
 |===
 |Arguments |Description
 
+|`--include` (string)
+|The file path to Kubernetes resource manifests, for example, Role, Subscription. These supplement or override defaults generated by run/cleanup.
+
+|`--install-mode` (string)
+|The `OperatorGroup` is created with the specified `InstallMode`. Format: `InstallModeType[=ns1,ns2[, ...]]``
+
 |`--kubeconfig` (string)
-|The file path to a Kubernetes configuration file. Default: `$HOME/.kube/config`
+|The file path to a Kubernetes configuration file. Defaults to the location specified by `$KUBECONFIG`, or to default file rules if not set
+
+|`--olm-namespace` (string)
+|The namespace where the OLM is installed. Default: `olm`.
 
-|`--local`
-|The Operator is run locally by building the Operator binary with the ability to access a Kubernetes cluster using a `kubeconfig` file.
+|`--operator-namespace` (string)
+|The namespace where the operator resources are created. The namespace must already exist in the cluster, or be defined in a manifest passed to `--include`.
```

Comment:
If you were able to revise the earlier instance, please revise here:
* "...or be defined in a manifest that is passed..."
* (maybe) "...defined in a manifest that is specified with the `--include` argument."